src/translation/ja-JP.tsx

```diff
@@ -131,17 +113,32 @@ export const Translation: TranslationComponent = (props) => {
 					コンテキスト関数は、コンテキストが既知の場合にのみ使用できます。
 				</Fragment>
 			);
-		case "definitionTooltip":
+
+		case "constructor":
+			return <Fragment>コンストラクタ</Fragment>;
+		case "constructorDescription":
+			return (
+				<Fragment>
+					型がコンストラクタを持っていれば、その型の新しい値を生成する関数のように呼び出すことができます。
```

Comment:
```suggestion
					コンストラクタを持つ型の場合、関数のように呼び出すことで、その型の新しい値を作成できます。
```

@typst-community/jp
Do you have any suggestions for a more natural translation?

Reply:
I've committed this version because it has been three days.
If you or anyone has a better idea, we can re-update it.